certified-connectors/XC-Gate/Readme.md

```diff
@@ -0,0 +1,32 @@
+# XC-Gate
```

Comment:
Hi @tec1-472, 

Can you please update readme.md file as per the template and make sure that all required fields are added. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Hello @vmanoharas 
Thanks for the review.
Modified according to the template.